src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExperimentalOptions.cs

```diff
@@ -17,6 +17,8 @@ internal sealed class ExperimentalOptions
 
     public const string OtlpRetryEnvVar = "OTEL_DOTNET_EXPERIMENTAL_OTLP_RETRY";
 
+    public const string OtlpDiskRetryDirectoryPathEnvVar = "OTEL_DOTNET_EXPERIMENTAL_OTLP_DISK_RETRY_DIRECTORY_PATH";
```

Comment:
If an app is designed to run in a platform-independent environment and wants to use a temporary location, the restriction to set the environment variable `OTEL_DOTNET_EXPERIMENTAL_OTLP_DISK_RETRY_DIRECTORY_PATH` can make things complex. Should we write to the temporary path if this environment variable is not set? We could rely on .NET API to get temp path `Path.GetTempPath`. Customers will use OTEL_DOTNET_EXPERIMENTAL_OTLP_RETRY to enable offline storage anyway. We could explain the behavior in the documentation along with this environment variable.

Reply:
Works for me. Its more simple for users who want minimal configuration for trying out this feature.

Reply:
Updated